torchgeo/models/resnet.py

```diff
@@ -22,46 +22,16 @@
 _zhu_xlab_transforms = AugmentationSequential(
     K.Resize(256),
     K.CenterCrop(224),
-    K.Normalize(mean=0, std=10000),
+    K.Normalize(mean=torch.tensor(0), std=torch.tensor(10000)),
     data_keys=["image"],
 )
 
 # https://github.com/ServiceNow/seasonal-contrast/blob/8285173ec205b64bc3e53b880344dd6c3f79fa7a/datasets/bigearthnet_dataset.py#L13 # noqa: E501
 _seco_transforms = AugmentationSequential(
     K.Resize(128),
     K.Normalize(
-        mean=torch.Tensor(
-            [
-                340.76769064,
-                429.9430203,
-                614.21682446,
-                590.23569706,
-                950.68368468,
-                1792.46290469,
-                2075.46795189,
-                2218.94553375,
-                2266.46036911,
-                2246.0605464,
-                1594.42694882,
-                1009.32729131,
-            ]
-        ),
-        std=torch.Tensor(
-            [
-                554.81258967,
-                572.41639287,
-                582.87945694,
-                675.88746967,
-                729.89827633,
-                1096.01480586,
-                1273.45393088,
-                1365.45589904,
-                1356.13789355,
-                1302.3292881,
-                1079.19066363,
-                818.86747235,
-            ]
-        ),
+        mean=torch.tensor([590.23569706, 614.21682446, 429.9430203]),
+        std=2 * torch.tensor([675.88746967, 582.87945694, 572.41639287]),
```

Comment:
SeCo and SSL4EO use 2x the std dev for normalization:

* https://github.com/ServiceNow/seasonal-contrast/blob/8285173ec205b64bc3e53b880344dd6c3f79fa7a/datasets/bigearthnet_dataset.py#L112,L113
* https://github.com/zhu-xlab/SSL4EO-S12/blob/d2868adfada65e40910bfcedfc49bc3b20df2248/src/benchmark/pretrain_ssl/datasets/SSL4EO/ssl4eo_dataset.py#L37,L38

Reply:
https://github.com/ServiceNow/seasonal-contrast/blob/8285173ec205b64bc3e53b880344dd6c3f79fa7a/datasets/bigearthnet_dataset.py#L111 Is this also different from the Kornia Normalize method, as they use some min max operation and also clip to range [0, 255]?

Reply:
True, this is actually quite different. They substract (mean - 2 * std), not mean, and they divide by (4 * std), not std. I have no idea why they scale by 255, seems odd unless you're trying to plot something. And we obviously aren't clamping (again, only useful for plotting). 

EDIT: oh, is it because they need to load it as a PIL image? Maybe that removes the 255 when converting to a Tensor.

Reply:
Not sure how they're even using PIL here since it only supports RGB, not MSI.

Reply:
(I guess that's why SeCo only had RGB weights?)

Reply:
Ah yes, they divide by 10K * 255 for S1 but not for S2?

Reply:
The more I look at these, the less sure I am that they are correct. May want to reach out to the authors.

Reply:
Once we figure this out, we should also fix the SeCO datamodule I recently added. According to @wangyi111, it actually doesn't matter that much what the exact normalization is.

Reply:
Opened an issue to get to the bottom of this. I actually think we're using the wrong normalization (this was only used for BigEarthNet, not for any other dataset). My guess is the SeCo normalization, but we'll see if we get a response. If I don't hear anything back, let's just assume SeCo normalization.

https://github.com/ServiceNow/seasonal-contrast/issues/19